stac_fastapi/extensions/tests/test_transaction.py

```diff
@@ -46,6 +47,19 @@ def update_item(self, collection_id: str, item_id: str, item: Item, **kwargs):
             "type": item.type,
         }
 
+    def patch_item(
+        self,
+        collection_id: str,
+        item_id: str,
+        patch: Union[PartialItem, List[PatchOperation]],
+        **kwargs,
+    ):
+        return {
```

Comment:
I think we should also test `PartialItem.merge_to_operations` here

Reply:
The conversion was missing of the initial `/` on the path so these tests were a good addition thanks.